src/set_theory/ordinal/notation.lean

```diff
@@ -582,7 +582,7 @@ theorem NF_repr_split' : ∀ {o o' m} [NF o], split' o = (o', m) → NF o' ∧ r
     cases NF_repr_split' h' with IH₁ IH₂,
     simp [IH₂, split'],
     intros, substs o' m,
-    have : ω ^ repr e = ω ^ (1 : ordinal.{0}) * ω ^ (repr e - 1),
+    have : ordinal.omega.{0} ^ repr e = ω ^ (1 : ordinal.{0}) * ω ^ (repr e - 1),
```

Comment:
Why do we lose the notation here? Is it because of the need for the universe variable? I think losing the `ω` notation here and elsewhere is a regression and it would be better to use this sort of solution instead:
```suggestion
    have : (ω : ordinal.{0}) ^ repr e = ω ^ (1 : ordinal.{0}) * ω ^ (repr e - 1),
```

Reply:
ω previously had type `ordinal.{0}` (making it almost unusable outside of this file), while it has now been made universe polymorphic. I'll go with your solution.

Reply:
There's also a few places where I need to use `aleph_0.{u}`, should that be changed to `(ℵ₀ : cardinal.{u})`?

Reply:
I think either is fine